src/collection/components/modals/CutFragmentModal.tsx

```diff
@@ -145,6 +145,10 @@ const CutFragmentModal: FunctionComponent<CutFragmentModalProps> = ({
 		onClose();
 	};
 
+	const startStartTimeOnce = once(() => {
+		setModalVideoSeekTime(fragmentStartTime);
+	});
```

Comment:
Maybe extract and reuse this function by both components?

Reply:
yes, i could extract this:
```
export const setModalSeekTimeOnce = () => once(() => {
		setModalVideoSeekTime(fragmentStartTime);
});
```

But then i still need to add the const in the component and import the function:
```
import { setModalSeekTimeOnce } from '...helpers';

const startStartTimeOnce = setModalSeekTimeOnce();
```

So this solution would be 7 lines of code
While the current solution is 6
So it isn't really worth it. but if the function was a little bigger, i would do it. 👍